app/test_main.py

```diff
@@ -1,3 +1,21 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_should_return_zero_if_age_less_than_first_year() -> None:
```

Comment:
Consider using `pytest.mark.parametrize` to simplify and organize your test cases. This will help avoid repetition and make your tests more maintainable, as suggested in the checklist.